homeassistant/components/zigbee.py

```diff
@@ -308,8 +307,7 @@ def handle_frame(frame):
         subscribe(hass, handle_frame)
 
         # Get initial state
-        hass.pool.add_job(
-            JobPriority.EVENT_STATE, (self.update_ha_state, True))
+        self.hass.loop.create_task(self.async_update_ha_state(True))
```

Comment:
We should not allow components to create tasks anymore because it will not be registered with hass